localstack/utils/aws/client_types.py

```diff
@@ -0,0 +1,194 @@
+import abc
+from typing import TYPE_CHECKING, Union
+
+if TYPE_CHECKING:
+    from mypy_boto3_acm import ACMClient
```

Comment:
Love that these are now separated into their own file. Will keep the churn here much better isolated :+1: